smokey/features/environment.py

```diff
@@ -36,10 +49,23 @@ def before_all(context):
     for k, v in CONFIG_DEFAULTS.items():
         context.config.userdata.setdefault(k, v)
 
+    # Load test user keys from environment
+    context.test_users = {}
+    for user in TEST_USERS:
+        token_key = 'TEST_USER_{user}_KEY'.format(user=user.upper())
+        userid_key = 'TEST_USER_{user}_USERID'.format(user=user.upper())
```

Comment:
Based on my own screwups in testing this, it might be a good idea to validate the format of the user ID here.